testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -157,21 +162,36 @@ def test_set_undefined_occupancy(self):
 
 
 # INVALID: cannot create Atoms in isolation like this;
-# will require refactor to build empty Universe and add atoms to it
+# will require refactor to build empty Universe and add atoms to it.
+# Also requires fixing implementation of some NotImplementedError
 @skip
 class TestAtomComparisons(TestCase):
+    def set_atom(self, index):
```

Comment:
would this be a way to go or do you have another idea in mind to create such test universes and AtomGroups?